### PR TITLE
blog post

### PR DESCRIPTION
**Pull request by BaileeKu**
I could not find two other posts by students in my class that were relevant to my topic. I did include three links, but you'll have to let me know whether or not they meet qualifications. They are what I found useful in my reading.

### CODE REVIEW

File: 2019-10-18-Bailee Ku/baileeku/_index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+bio: The author, a student at Florida State University, was enrolled in the digital microhistory lab in fall 2019.
+
+interests: 
+
+education:
+
+social: BaileeKu
+- icon: github
+  icon_pack: fab
+  link: https://github.com/
```
Comment (whanley) [inline 2019-10-18-Bailee Ku/baileeku/_index.md]: add github username to link

**Review by whanley — COMMENTED** [review]
Great find. This is an important book. A few things:
- in Markdown, use double return to indicate paragraph breaks.
- this book was not a novel. Novels are fiction. This was (purportedly) non-fiction.
- Blunt was a great adversary of Cromer, whom I've mentioned several times. Google "Blunt Cromer" and read a bit of background. 
- You can find this book online. Take a look and see what you think! Compare the contents to what the reviewer chooses to write about.
- I like your writing style.

**Comment by whanley** [comment]
Please respond to comments.

**Comment by BaileeKu** [comment]
Okay, I'll finish up the blog post over break!

**Pull request merged**